lib/rubocop/cop/rspec/predicate_matcher.rb

```diff
@@ -149,12 +161,25 @@ def check_explicit(node) # rubocop:disable Metrics/MethodLength
           return if part_of_ignored_node?(node)
 
           predicate_matcher?(node) do |actual, matcher|
+            next unless replaceable_matcher?(matcher)
+
             add_offense(node, message: message_explicit(matcher)) do |corrector|
               corrector_explicit(corrector, node, actual, matcher, matcher)
             end
           end
         end
 
+        def replaceable_matcher?(matcher)
+          case matcher.method_name.to_s
+          when 'be_a', 'be_an', 'be_a_kind_of', 'a_kind_of', 'be_kind_of',
+            'be_an_instance_of', 'be_instance_of', 'an_instance_of',
+            'include', 'respond_to', /^have_(.+)/
```

Comment:
Ditto:

```suggestion
            'include', 'respond_to', /\Ahave_(.+)/
```

Reply:
It looks like the capture (`(.+)`) is not used for anything. So I think it could be replaced by a dot:
```suggestion
            'include', 'respond_to', /\Ahave_./
```

Reply:
Thank you so much. I updated this PR.